frontend/src/app/modules/work_packages/routing/wp-view-base/work-package-single-view.base.ts

```diff
@@ -143,6 +143,7 @@ export class WorkPackageSingleViewBase extends UntilDestroyedMixin {
    * Recompute the current tab focus label
    */
   public updateFocusAnchorLabel(tabName:string):string {
+    // TODO: what to do there when tabName is "additionalTabs" where do i get the value which tab I have at hand? -> ask oliver
```

Comment:
This seems to be some a11y-related feature (right?) - the problem is that `tabName` is just `additionalTabs` here and I can't figure out which tab exactly we show - would be great if I could reach the tab identifier here. Maybe that's another pair-debugging session with @oliverguenther 🙏

Reply:
Sure, we'll take a look at it together later